### PR TITLE
[IMP] portal, web_editor, website, *: fix python/js linter errors 

### PR DESCRIPTION
**Pull request by rdeodoo**
*: http_routing, website_blog

Let's clean the linter error once and for all:
- It's really annoying to have the linter yelling when opening a file
- It's bad because people do copy paste code lines, ultimatelly copying
  linter errors
- For unused imports, it's a real problem as when you touch a line next
  to it or remove one element from the line, it raises a `ci-style` on
  the line/element you did not even touch.
  You then have to either bypass the `ci-style` or remove the unused
  import which is not related to your diff.

### CODE REVIEW

**Comment by rdeodoo** [comment]
@qsm-odoo Before I rebase and squash/rewrite commits, anything against this change?
I've ensured it won't slow down @detrouxdev owl conversion work.

**Comment by qsm-odoo** [comment]
@rdeodoo Of course not

**Comment by rdeodoo** [comment]
@qsm-odoo :+1: 

@robodoo delegate=@bso-odoo

File: addons/web_editor/models/ir_qweb_fields.py [hunk 1]
```diff
@@ -20,11 +20,9 @@
 import requests
 from datetime import datetime
 from lxml import etree, html
-from PIL import Image as I
+from PIL import Image as Img  # 'Image' is already a class name in this file
```
Comment (rdeodoo) [inline addons/web_editor/models/ir_qweb_fields.py:23]: @odoo/rd-security This one is triggering the `ci/security` (for the 2 lines usage below).
Nothing dangerous, just a renaming.

File: addons/website/tests/test_page.py [hunk 2]
```diff
@@ -283,7 +283,7 @@ def test_03_error_page_debug(self):
             self.assertIn('ZeroDivisionError: division by zero', r.text, "Error should be shown in debug.")
 
     def test_04_visitor_no_session(self):
-        with patch.object(root.session_store, 'save') as session_save,\
+        with patch.object(root.session_store, 'save') as session_save, \
```
Comment (bso-odoo) [inline addons/website/tests/test_page.py:286]: On the python-side, this is the only issue fixed in this PR that I could not get triggered by a linter rule. Do you know which rule was infringed on your side ?

**Review by xmo-odoo — COMMENTED** [review]
@robodoo override=ci/security

File: addons/website/tests/test_page.py (hunk 2 above)
Comment (rdeodoo) [reply addons/website/tests/test_page.py:286]: image
E231 it seems

**Comment by rdeodoo** [comment]
There is multiple reasons to do it:
- People **do** copy paste, spreading the errors
- You sometime have to force a `ci-style` override or add unrelated diff to your PR because of unused import
- It's a nightmare in some cases to edit a file, because of those many linter errors in some "sapin de noel" files, it's hard to see your own errors should you make one. See:

| py | js | 
| -- | -- |
| image | image |

We don't want to keep this kind of code forever, the sooner it's cleaned the better.
I also only fixed the modules we are in charge of (except `web_editor` JS files, there is too many from the framework-editor team, those files are shared between our teams) to not impact other teams.

Also, it shouldn't be that problematic regarding forward-port, it's not like this code is often changed in stable, and it's mostly one liners, chance is low.

But I get your point, it's why we do this kind of change very rarely. Now was a good time to get rid of those.

Finally, not 100% sure, but some of those are there since a long time and should not be allowed anymore by the runbot, once it's cleaned, it shouldn't be able to reintroduce them.

**Pull request closed without merging**